deliver/spec/upload_screenshots_spec.rb

```diff
@@ -14,7 +14,7 @@
                               get_app_screenshot_sets: [app_screenshot_set])
         screenshots_per_language = { 'en-US' => [] }
 
-        # emurate Deliver::AppScreenshotIterator#each_app_screenshot_set's two behaviors with or without given block
+        # emulate Deliver::AppScreenshotIterator#each_app_screenshot_set's two behaviors with or without given block
```

Comment:
This entire update pass was done manually. I suspect when I did the previous pass I wasn't sure about certain items, like this. I'm going out on a limb and guessing this is right, but I could be wrong.